pages/Hypr Ecosystem/hyprpaper.md

```diff
@@ -5,6 +5,31 @@ title: hyprpaper
 
 hyprpaper is a fast, IPC-controlled wallpaper utility for Hyprland.
 
+## Installation
+
+### [Arch Linux Install](https://archlinux.org/packages/extra/x86_64/hyprpaper/)
+
+```sh
+pacman -S hyprpaper
+```
+
+### [OpenSuse Linux Install](https://software.opensuse.org/package/hyprpaper)
+
+```sh
+zypper install hyprpaper
+```
+
+### Manual
+
+For manual installation please check the [docs](https://github.com/hyprwm/hyprpaper?tab=readme-ov-file#manual)
```

Comment:
For manual installation decided to link to the repo mainly beacuse copying the whole description and paste it to the page would leave in a very large section compared to the rest.
IMO regardless of the size of the section it would be easy to paste the whole explanation here :)